### DISCUSSION TITLE
Side-by-side comparison in playground

### DISCUSSION THREAD

**Discussion by simonwh** [langfuse/langfuse#4483] [Ideas]
### Describe the feature or potential improvement

To be able to compare multiple models and/or prompts side by side in the playground.

This would make it so much easier when you are in the first phase of prompt engineering, where you are typically experimenting with prompts and want to grasp what's possible with different models, before you start building datasets and relying on experiments.

**Comment by marcklingen** [comment]
Thanks for sharing! We plan to bring [prompt experiments](https://langfuse.com/docs/datasets/prompt-experiments) into the playground to allow for side-by-side experiments. Will keep you in the loop here.

**Comment by marcklingen** [comment]
Changelog: https://langfuse.com/changelog/2025-07-28-playground-side-by-side
